street_network/src/lanes/mod.rs

```diff
@@ -154,6 +172,8 @@ impl LaneType {
             LaneType::Construction => 'x',
             LaneType::LightRail => 'l',
             LaneType::Buffer(_) => '|',
+            LaneType::Footway => 'f',
+            LaneType::SharedUse => 'V',
```

Comment:
There was not an obvious choice here :(

Reply:
Large '`F`'? It is appropriate that is suggests footway, I think.